src/main/java/jenkins/plugins/jobcacher/CacheWrapper.java

```diff
@@ -92,12 +92,12 @@ public void setSkipSave(boolean skipSave) {
         this.skipSave = skipSave;
     }
 
+    @DataBoundSetter
     @SuppressWarnings("unused")
     public void setSkipRestore(boolean skipRestore) {
         this.skipRestore = skipRestore;
     }
 
-    @DataBoundSetter
     @SuppressWarnings("unused")
     public boolean setSkipRestore() {
```

Comment:
Shouldn't this be `getSkipRestore` ?

Reply:
Yea... There was some mix-up with getter/setter.

I've reactivated the test and should fix the issue on https://github.com/jenkinsci/jobcacher-plugin/pull/274